spec.bs

```diff
@@ -3050,45 +3056,65 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   : <dfn>is chaff</dfn>
   :: A [=boolean=] field indicating whether this response should be ignored.
   : <dfn>ad render url</dfn>
-  :: The [=URL=] of the ad for the winning bid.
+  :: [=URL=]. The [=leading bid info/leading bid=]'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] from the
+     component auction executed on the trusted auction server.
   : <dfn>ad components</dfn>
-  :: A [=list=] of [=URLs=] containing the ad components for the winning bid.
+  :: A [=list=] of [=URLs=]. A list of [=leading bid info/leading bid=]'s
+     [=generated bid/ad component descriptors=]'s [=ad descriptor/urls=] from the
+     component auction executed on the trusted auction server.

```

Comment:
only available from component auctions? Same to a few other fields below.

Reply:
Done.